src/lib/index.jsx

```diff
@@ -105,6 +107,7 @@ class SingleOtpInput extends PureComponent<*> {
             hasErrored && errorStyle
           )}
           type={isInputNum ? 'tel' : 'text'}
+          type={isSecure ? 'password' : 'text'}
```

Comment:
Since this prop will always override the `type` being set above, it might be a better idea to consolidate them into one, for e.g., with a function.

Reply:
Ok, will create a new PR with required changes